pkg/services/authn/clients/api_key.go

```diff
@@ -19,9 +19,10 @@ import (
 )
 
 var (
-	errAPIKeyInvalid = errutil.Unauthorized("api-key.invalid", errutil.WithPublicMessage("Invalid API key"))
-	errAPIKeyExpired = errutil.Unauthorized("api-key.expired", errutil.WithPublicMessage("Expired API key"))
-	errAPIKeyRevoked = errutil.Unauthorized("api-key.revoked", errutil.WithPublicMessage("Revoked API key"))
+	errAPIKeyInvalid    = errutil.Unauthorized("api-key.invalid", errutil.WithPublicMessage("Invalid API key"))
+	errAPIKeyExpired    = errutil.Unauthorized("api-key.expired", errutil.WithPublicMessage("Expired API key"))
+	errAPIKeyRevoked    = errutil.Unauthorized("api-key.revoked", errutil.WithPublicMessage("Revoked API key"))
+	errAPIKeyInvalidOrg = errutil.Unauthorized("api-key.invalid-organization", errutil.WithPublicMessage("API key does not belong to the requested organization"))
```

Comment:
Any suggestion?

Reply:
Maybe 
```suggestion
	errAPIKeyInvalidOrg = errutil.Unauthorized("api-key.organization-mismatch", errutil.WithPublicMessage("API key does not belong to the requested organization"))
```
or something like that?

But the current error also looks fine to me.